tendermint/src/account.rs

```diff
@@ -15,7 +15,7 @@ use subtle_encoding::hex;
 const LENGTH: usize = 20;
 
 /// Account IDs
-#[derive(Copy, Clone, Eq, Hash, PartialEq, PartialOrd, Ord)]
+#[derive(Copy, Clone, Eq, Hash, PartialEq, PartialOrd, Ord, Default)]
```

Comment:
For anything cryptographic / security-critical, I don't think it makes sense to add a `Default`  impl which initializes it to what's effectively an invalid value solely for the purposes of testing.

If your goal with this is to do something like `Id::default()` for tests, note that you could also do `Id::new(Default::default())` to get the same effect.

Reply:
Yeah, kinda agree with you, this use of  ``Default`` trait is purely due to laziness ;D
I think I should just create some custom data templates for tests.